zebedee-cms/src/main/java/com/github/onsdigital/zebedee/verification/http/PooledHttpClient.java

```diff
@@ -164,7 +164,8 @@ private URI buildPath(String path) {
 
     private URIBuilder newUriBuilder(String path) {
         URIBuilder uriBuilder = new URIBuilder(HOST);
-        uriBuilder.setPath((uriBuilder.getPath() + "/" + path).replaceAll("//+", "/"));
+        String uri = StringUtils.defaultIfEmpty(uriBuilder.getPath(), "");
```

Comment:
If `uriBuilder.getPath()` returns `null` then `uriBuilder.getPath() + "/"` results in `"null/"`.

Reply:
looks good to me but do you want a more qualified eye to look at it?